modules.json

```diff
@@ -386,6 +386,15 @@
        "chayim"
     ],
     "stars":0
-  }
-
+  },
+  {
+    "name": "redis-interval-module",
+    "license": "MIT",
+    "repository": "https://github.com/ogama/redis-interval-module",
+    "description": "Redis module for creation and manipulation of interval set.",
```

Comment:
interval sets*

Reply:
I agree, but it's not very important, so I decided to merge as-is.

Reply:
Sure, I agree